stories/alien/level1/slides.html

```diff
@@ -490,48 +552,58 @@
 -The basic idea of nanopore sequencing is that in the channel of nanopore there is not so much space.
 -Therefore, how much exactly the DNA blocks this channel depends on the nucleotide sequence.
 
+
 ---
 
 ### Sequencing
 
-![https://streetscience.community/protocols/beer-dna-sequencing/#sequencing](images/nanopore-seq-process.png)
+![Sam on the left side of DNA](images/nanopore-seq-process.png)
 
 <small><small>[https://streetscience.community/protocols/beer-dna-sequencing/#sequencing]</small></small>
 
 ???
--In this picture we can see the scheme of nanopore sequencing.
--Sam decided to participate in this process in order to learn more about how it works.
+-Here you can see the scheme of nanopore sequencing.
+-Sam decides to participate in this process in order to learn more about how it works.
 -As Sam is a tiny guy it was not a problem.
 
 ---
 
 ### Sequencing
+
 ![Sam_on_DNA](images/Sam_on_DNA.png)
 
+<small><small>[https://streetscience.community/protocols/beer-dna-sequencing/#sequencing]</small></small>
+
 ???
--Sam hooked on DNA. 
+-Sam hooks on to DNA. 
+
 ---
 
 ### Sequencing
 ![Sam_on_motor](images/Sam_on_motor.png)
```

Comment:
I would still split the picture so that only one pore is viable.

Reply:
> I would still split the picture so that only one pore is viable.

done